packages/react-magma-dom/src/components/Stepper/Stepper.stories.tsx

```diff
@@ -151,7 +175,38 @@ const Template: Story<StepperProps> = args => {
           </ButtonGroup>
         </Flex>
       </Flex>
-    </>
+    </ResponsibleStepperContainer>
+  );
+};
+
+const InsideDropdownTemplate: Story<StepperProps> = args => {
```

Comment:
I noticed that in this example when we set the orientation to horizontal and hide labels inside a dropdown, this happens:
![image](https://github.com/user-attachments/assets/a8cd6569-77a3-47ee-ac2a-eac48738ea91)

I think we may have to force a width for the dropdown content or stepper for this to look as expected